src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Strings.resx

```diff
@@ -135,4 +135,7 @@
   <data name="GlobalJsonResolutionFailed" xml:space="preserve">
     <value>Unable to locate the .NET SDK as specified by global.json, please check that the specified version is installed.</value>
   </data>
+  <data name="GlobalJsonResolutionFailedSpecificVersion" xml:space="preserve">
+    <value>Unable to locate the .NET SDK version '{0}' as specified by global.json, please check that the specified version is installed.</value>
```

Comment:
Looks great, thanks for doing this. If I can be greedy for a moment since you're in here already - can you incorporate the path to the global.json file that caused this resolution result (which should be present in `resolverResult.GlobalJsonPath`) into the error message as well?

```
Unable to locate the .NET SDK version '{0}' as specified by the global.json at {1}, please check that the specified version is installed.
```

And similar for the `GlobalJsonResolutionFailed` error message directly above this?

Reply:
With the second variable, there are now four cases to consider:

1. Unknown global.json path, unknown version
2. Known path, unknown version
3. Unknown path, known version
4. Known path, known version

The easiest option is to use the new message only for case (4), and the only case which is penalized from such a change is case (3) since it would no longer show the expected version. Will that be a problem?